src/scripts/browserDeviceProfile.js

```diff
@@ -659,6 +677,14 @@ export default function (options) {
         webmVideoCodecs.push('vp8');
     }
 
+    // WebM support is partial/unreliable from Safari versions 17.0-17.4. It was fixed in 17.4:
+    //  * https://developer.apple.com/documentation/safari-release-notes/safari-17_4-release-notes
+    const safariSupportsWebm = browser.safari && (false // eslint-disable-line sonarjs/no-redundant-boolean
```

Comment:
Unclear why Sonarlint doesn't pick up this lint suppression. This is following the same code-style as the rest of the file.

Reply:
Iirc, it was manually rejected.